`-rdynamic` being used in compile phase inated of link phase

```
ninja
[2/457] Building C object kissfft/CMakeFiles/kissfft.dir/kiss_fftnd.c.o
/home/solomoncyj/Downloads/linux-wallpaperengine/src/External/kissfft/kiss_fftnd.c:50:10: warning: cast from 'char *' to 'kiss_fftnd_cfg' (aka 'struct kiss_fftnd_state *') increases required alignment from 1 to 8 [-Wcast-align]
   50 |     st = (kiss_fftnd_cfg) ptr;
      |          ^~~~~~~~~~~~~~~~~~~~
/home/solomoncyj/Downloads/linux-wallpaperengine/src/External/kissfft/kiss_fftnd.c:55:18: warning: cast from 'char *' to 'kiss_fft_cfg *' (aka 'struct kiss_fft_state **') increases required alignment from 1 to 8 [-Wcast-align]
   55 |     st->states = (kiss_fft_cfg *)ptr;
      |                  ^~~~~~~~~~~~~~~~~~~
/home/solomoncyj/Downloads/linux-wallpaperengine/src/External/kissfft/kiss_fftnd.c:58:16: warning: cast from 'char *' to 'int *' increases required alignment from 1 to 4 [-Wcast-align]
   58 |     st->dims = (int*)ptr;
      |                ^~~~~~~~~
/home/solomoncyj/Downloads/linux-wallpaperengine/src/External/kissfft/kiss_fftnd.c:61:18: warning: cast from 'char *' to 'kiss_fft_cpx *' increases required alignment from 1 to 4 [-Wcast-align]
   61 |     st->tmpbuf = (kiss_fft_cpx*)ptr;
      |                  ^~~~~~~~~~~~~~~~~~
4 warnings generated.
[3/457] Building C object kissfft/CMakeFiles/kissfft.dir/kiss_fftndr.c.o
/home/solomoncyj/Downloads/linux-wallpaperengine/src/External/kissfft/kiss_fftndr.c:59:10: warning: cast from 'char *' to 'kiss_fftndr_cfg' (aka 'struct kiss_fftndr_state *') increases required alignment from 1 to 8 [-Wcast-align]
   59 |     st = (kiss_fftndr_cfg) ptr;
      |          ^~~~~~~~~~~~~~~~~~~~~
1 warning generated.
[4/457] Building C object kissfft/CMakeFiles/kissfft.dir/kiss_fftr.c.o
/home/solomoncyj/Downloads/linux-wallpaperengine/src/External/kissfft/kiss_fftr.c:49:18: warning: cast from 'char *' to 'kiss_fft_cpx *' increases required alignment from 1 to 4 [-Wcast-align]
   49 |     st->tmpbuf = (kiss_fft_cpx *) (((char *) st->substate) + subsize);
      |                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 warning generated.
[6/457] Building CXX object libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_atomic_flag.cc.o
FAILED: [code=1] libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_atomic_flag.cc.o 
/usr/local/bin/ccache  /usr/bin/clang++ -DCEF_USE_SANDBOX -DWRAPPING_CEF_SHARED -D_FILE_OFFSET_BITS=64 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -Dlibcef_dll_wrapper_EXPORTS -I/home/solomoncyj/Downloads/linux-wallpaperengine/build/cef/cef_binary_135.0.17+gcbc1c5b+chromium-135.0.7049.52_linux64_minimal -march=native -flto=thin -Wno-narrowing -Os -DNDEBUG -std=gnu++20 -fPIC -fno-strict-aliasing -fPIC -fstack-protector -funwind-tables --param=ssp-buffer-size=4 -pipe -pthread -Wall -Werror -Wno-missing-field-initializers -Wno-unused-parameter -Wno-error=comment -Wno-comment -Wno-deprecated-declarations -m64 -march=x86-64 -rdynamic -fno-threadsafe-statics -fvisibility-inlines-hidden -Wsign-compare -Wno-undefined-var-template -Wno-narrowing -std=c++20 -MD -MT libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_atomic_flag.cc.o -MF libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_atomic_flag.cc.o.d -o libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_atomic_flag.cc.o -c /home/solomoncyj/Downloads/linux-wallpaperengine/build/cef/cef_binary_135.0.17+gcbc1c5b+chromium-135.0.7049.52_linux64_minimal/libcef_dll/base/cef_atomic_flag.cc
clang++: error: argument unused during compilation: '-rdynamic' [-Werror,-Wunused-command-line-argument]
[7/457] Building CXX object libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/transfer_util.cc.o
FAILED: [code=1] libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/transfer_util.cc.o 
/usr/local/bin/ccache  /usr/bin/clang++ -DCEF_USE_SANDBOX -DWRAPPING_CEF_SHARED -D_FILE_OFFSET_BITS=64 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -Dlibcef_dll_wrapper_EXPORTS -I/home/solomoncyj/Downloads/linux-wallpaperengine/build/cef/cef_binary_135.0.17+gcbc1c5b+chromium-135.0.7049.52_linux64_minimal -march=native -flto=thin -Wno-narrowing -Os -DNDEBUG -std=gnu++20 -fPIC -fno-strict-aliasing -fPIC -fstack-protector -funwind-tables --param=ssp-buffer-size=4 -pipe -pthread -Wall -Werror -Wno-missing-field-initializers -Wno-unused-parameter -Wno-error=comment -Wno-comment -Wno-deprecated-declarations -m64 -march=x86-64 -rdynamic -fno-threadsafe-statics -fvisibility-inlines-hidden -Wsign-compare -Wno-undefined-var-template -Wno-narrowing -std=c++20 -MD -MT libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/transfer_util.cc.o -MF libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/transfer_util.cc.o.d -o libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/transfer_util.cc.o -c /home/solomoncyj/Downloads/linux-wallpaperengine/build/cef/cef_binary_135.0.17+gcbc1c5b+chromium-135.0.7049.52_linux64_minimal/libcef_dll/transfer_util.cc
clang++: error: argument unused during compilation: '-rdynamic' [-Werror,-Wunused-command-line-argument]
[8/457] Building CXX object libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/shutdown_checker.cc.o
FAILED: [code=1] libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/shutdown_checker.cc.o 
/usr/local/bin/ccache  /usr/bin/clang++ -DCEF_USE_SANDBOX -DWRAPPING_CEF_SHARED -D_FILE_OFFSET_BITS=64 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -Dlibcef_dll_wrapper_EXPORTS -I/home/solomoncyj/Downloads/linux-wallpaperengine/build/cef/cef_binary_135.0.17+gcbc1c5b+chromium-135.0.7049.52_linux64_minimal -march=native -flto=thin -Wno-narrowing -Os -DNDEBUG -std=gnu++20 -fPIC -fno-strict-aliasing -fPIC -fstack-protector -funwind-tables --param=ssp-buffer-size=4 -pipe -pthread -Wall -Werror -Wno-missing-field-initializers -Wno-unused-parameter -Wno-error=comment -Wno-comment -Wno-deprecated-declarations -m64 -march=x86-64 -rdynamic -fno-threadsafe-statics -fvisibility-inlines-hidden -Wsign-compare -Wno-undefined-var-template -Wno-narrowing -std=c++20 -MD -MT libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/shutdown_checker.cc.o -MF libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/shutdown_checker.cc.o.d -o libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/shutdown_checker.cc.o -c /home/solomoncyj/Downloads/linux-wallpaperengine/build/cef/cef_binary_135.0.17+gcbc1c5b+chromium-135.0.7049.52_linux64_minimal/libcef_dll/shutdown_checker.cc
clang++: error: argument unused during compilation: '-rdynamic' [-Werror,-Wunused-command-line-argument]
[9/457] Building CXX object libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_dump_without_crashing.cc.o
FAILED: [code=1] libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_dump_without_crashing.cc.o 
/usr/local/bin/ccache  /usr/bin/clang++ -DCEF_USE_SANDBOX -DWRAPPING_CEF_SHARED -D_FILE_OFFSET_BITS=64 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -Dlibcef_dll_wrapper_EXPORTS -I/home/solomoncyj/Downloads/linux-wallpaperengine/build/cef/cef_binary_135.0.17+gcbc1c5b+chromium-135.0.7049.52_linux64_minimal -march=native -flto=thin -Wno-narrowing -Os -DNDEBUG -std=gnu++20 -fPIC -fno-strict-aliasing -fPIC -fstack-protector -funwind-tables --param=ssp-buffer-size=4 -pipe -pthread -Wall -Werror -Wno-missing-field-initializers -Wno-unused-parameter -Wno-error=comment -Wno-comment -Wno-deprecated-declarations -m64 -march=x86-64 -rdynamic -fno-threadsafe-statics -fvisibility-inlines-hidden -Wsign-compare -Wno-undefined-var-template -Wno-narrowing -std=c++20 -MD -MT libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_dump_without_crashing.cc.o -MF libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_dump_without_crashing.cc.o.d -o libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_dump_without_crashing.cc.o -c /home/solomoncyj/Downloads/linux-wallpaperengine/build/cef/cef_binary_135.0.17+gcbc1c5b+chromium-135.0.7049.52_linux64_minimal/libcef_dll/base/cef_dump_without_crashing.cc
clang++: error: argument unused during compilation: '-rdynamic' [-Werror,-Wunused-command-line-argument]
[10/457] Building CXX object libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_callback_helpers.cc.o
FAILED: [code=1] libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_callback_helpers.cc.o 
/usr/local/bin/ccache  /usr/bin/clang++ -DCEF_USE_SANDBOX -DWRAPPING_CEF_SHARED -D_FILE_OFFSET_BITS=64 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -Dlibcef_dll_wrapper_EXPORTS -I/home/solomoncyj/Downloads/linux-wallpaperengine/build/cef/cef_binary_135.0.17+gcbc1c5b+chromium-135.0.7049.52_linux64_minimal -march=native -flto=thin -Wno-narrowing -Os -DNDEBUG -std=gnu++20 -fPIC -fno-strict-aliasing -fPIC -fstack-protector -funwind-tables --param=ssp-buffer-size=4 -pipe -pthread -Wall -Werror -Wno-missing-field-initializers -Wno-unused-parameter -Wno-error=comment -Wno-comment -Wno-deprecated-declarations -m64 -march=x86-64 -rdynamic -fno-threadsafe-statics -fvisibility-inlines-hidden -Wsign-compare -Wno-undefined-var-template -Wno-narrowing -std=c++20 -MD -MT libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_callback_helpers.cc.o -MF libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_callback_helpers.cc.o.d -o libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_callback_helpers.cc.o -c /home/solomoncyj/Downloads/linux-wallpaperengine/build/cef/cef_binary_135.0.17+gcbc1c5b+chromium-135.0.7049.52_linux64_minimal/libcef_dll/base/cef_callback_helpers.cc
clang++: error: argument unused during compilation: '-rdynamic' [-Werror,-Wunused-command-line-argument]
[11/457] Building CXX object libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_lock.cc.o
FAILED: [code=1] libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_lock.cc.o 
/usr/local/bin/ccache  /usr/bin/clang++ -DCEF_USE_SANDBOX -DWRAPPING_CEF_SHARED -D_FILE_OFFSET_BITS=64 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -Dlibcef_dll_wrapper_EXPORTS -I/home/solomoncyj/Downloads/linux-wallpaperengine/build/cef/cef_binary_135.0.17+gcbc1c5b+chromium-135.0.7049.52_linux64_minimal -march=native -flto=thin -Wno-narrowing -Os -DNDEBUG -std=gnu++20 -fPIC -fno-strict-aliasing -fPIC -fstack-protector -funwind-tables --param=ssp-buffer-size=4 -pipe -pthread -Wall -Werror -Wno-missing-field-initializers -Wno-unused-parameter -Wno-error=comment -Wno-comment -Wno-deprecated-declarations -m64 -march=x86-64 -rdynamic -fno-threadsafe-statics -fvisibility-inlines-hidden -Wsign-compare -Wno-undefined-var-template -Wno-narrowing -std=c++20 -MD -MT libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_lock.cc.o -MF libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_lock.cc.o.d -o libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_lock.cc.o -c /home/solomoncyj/Downloads/linux-wallpaperengine/build/cef/cef_binary_135.0.17+gcbc1c5b+chromium-135.0.7049.52_linux64_minimal/libcef_dll/base/cef_lock.cc
clang++: error: argument unused during compilation: '-rdynamic' [-Werror,-Wunused-command-line-argument]
[12/457] Building CXX object libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_callback_internal.cc.o
FAILED: [code=1] libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_callback_internal.cc.o 
/usr/local/bin/ccache  /usr/bin/clang++ -DCEF_USE_SANDBOX -DWRAPPING_CEF_SHARED -D_FILE_OFFSET_BITS=64 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -Dlibcef_dll_wrapper_EXPORTS -I/home/solomoncyj/Downloads/linux-wallpaperengine/build/cef/cef_binary_135.0.17+gcbc1c5b+chromium-135.0.7049.52_linux64_minimal -march=native -flto=thin -Wno-narrowing -Os -DNDEBUG -std=gnu++20 -fPIC -fno-strict-aliasing -fPIC -fstack-protector -funwind-tables --param=ssp-buffer-size=4 -pipe -pthread -Wall -Werror -Wno-missing-field-initializers -Wno-unused-parameter -Wno-error=comment -Wno-comment -Wno-deprecated-declarations -m64 -march=x86-64 -rdynamic -fno-threadsafe-statics -fvisibility-inlines-hidden -Wsign-compare -Wno-undefined-var-template -Wno-narrowing -std=c++20 -MD -MT libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_callback_internal.cc.o -MF libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_callback_internal.cc.o.d -o libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_callback_internal.cc.o -c /home/solomoncyj/Downloads/linux-wallpaperengine/build/cef/cef_binary_135.0.17+gcbc1c5b+chromium-135.0.7049.52_linux64_minimal/libcef_dll/base/cef_callback_internal.cc
clang++: error: argument unused during compilation: '-rdynamic' [-Werror,-Wunused-command-line-argument]
[13/457] Building CXX object libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_lock_impl.cc.o
FAILED: [code=1] libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_lock_impl.cc.o 
/usr/local/bin/ccache  /usr/bin/clang++ -DCEF_USE_SANDBOX -DWRAPPING_CEF_SHARED -D_FILE_OFFSET_BITS=64 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -Dlibcef_dll_wrapper_EXPORTS -I/home/solomoncyj/Downloads/linux-wallpaperengine/build/cef/cef_binary_135.0.17+gcbc1c5b+chromium-135.0.7049.52_linux64_minimal -march=native -flto=thin -Wno-narrowing -Os -DNDEBUG -std=gnu++20 -fPIC -fno-strict-aliasing -fPIC -fstack-protector -funwind-tables --param=ssp-buffer-size=4 -pipe -pthread -Wall -Werror -Wno-missing-field-initializers -Wno-unused-parameter -Wno-error=comment -Wno-comment -Wno-deprecated-declarations -m64 -march=x86-64 -rdynamic -fno-threadsafe-statics -fvisibility-inlines-hidden -Wsign-compare -Wno-undefined-var-template -Wno-narrowing -std=c++20 -MD -MT libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_lock_impl.cc.o -MF libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_lock_impl.cc.o.d -o libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_lock_impl.cc.o -c /home/solomoncyj/Downloads/linux-wallpaperengine/build/cef/cef_binary_135.0.17+gcbc1c5b+chromium-135.0.7049.52_linux64_minimal/libcef_dll/base/cef_lock_impl.cc
clang++: error: argument unused during compilation: '-rdynamic' [-Werror,-Wunused-command-line-argument]
[15/457] Building CXX object libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_logging.cc.o
FAILED: [code=1] libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_logging.cc.o 
/usr/local/bin/ccache  /usr/bin/clang++ -DCEF_USE_SANDBOX -DWRAPPING_CEF_SHARED -D_FILE_OFFSET_BITS=64 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -Dlibcef_dll_wrapper_EXPORTS -I/home/solomoncyj/Downloads/linux-wallpaperengine/build/cef/cef_binary_135.0.17+gcbc1c5b+chromium-135.0.7049.52_linux64_minimal -march=native -flto=thin -Wno-narrowing -Os -DNDEBUG -std=gnu++20 -fPIC -fno-strict-aliasing -fPIC -fstack-protector -funwind-tables --param=ssp-buffer-size=4 -pipe -pthread -Wall -Werror -Wno-missing-field-initializers -Wno-unused-parameter -Wno-error=comment -Wno-comment -Wno-deprecated-declarations -m64 -march=x86-64 -rdynamic -fno-threadsafe-statics -fvisibility-inlines-hidden -Wsign-compare -Wno-undefined-var-template -Wno-narrowing -std=c++20 -MD -MT libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_logging.cc.o -MF libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_logging.cc.o.d -o libcef_dll_wrapper/CMakeFiles/libcef_dll_wrapper.dir/base/cef_logging.cc.o -c /home/solomoncyj/Downloads/linux-wallpaperengine/build/cef/cef_binary_135.0.17+gcbc1c5b+chromium-135.0.7049.52_linux64_minimal/libcef_dll/base/cef_logging.cc
clang++: error: argument unused during compilation: '-rdynamic' [-Werror,-Wunused-command-line-argument]
[23/457] Building C object quickjs/CMakeFiles/qjs.dir/quickjs.c.o
ninja: build stopped: subcommand failed.
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -122,7 +122,6 @@ list(REMOVE_ITEM CEF_CXX_COMPILER_FLAGS -std=c++17)
 list(REMOVE_ITEM CEF_COMPILER_FLAGS -fvisibility=hidden)
 list(REMOVE_ITEM CEF_COMPILER_FLAGS -fvisibility-inlines-hidden)
 list(APPEND CEF_CXX_COMPILER_FLAGS -std=c++20)
-list(APPEND CEF_COMPILER_FLAGS -rdynamic)
 list(APPEND CEF_EXE_LINKER_FLAGS -rdynamic)
 # remove the vulkan library from the files to copy so it doesn't fail after removing it
 list(REMOVE_ITEM CEF_BINARY_FILES libvulkan.so.1)

